docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+- a proper recognition of the achieved work of any contributor,
+- compliance with the FAIR principles for Research Software (`FAIR4RS <https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs>`_).
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Reminder of the citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This scheme and these two references are precise on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors, or `contributors`, that requires to be updated with time according to a
+general policy. This section defines what is a contributor to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and, de-facto, the members of the Coordination Committee.
+
```

Comment:
Since the contribution policy is quite liberal, maybe people doing PR reviews can also be counted?

Reply:
That is indeed a very useful contribution to any open-source code.

Reply:
I agree, however technically I have no good proposal of how to handle this in practice. How would we maintain a list of reviewers, that are not code contributors and do not have a git record?

Reply:
This is my main issue, to identify people doing the review. However, concretely, the people making the reviews are at >99.9%:
- the Lead Dev, that make PR so they are in
- the core dev team, that make PR so they are in
Do we have reviewers that never made a PR???

Reply:
> Do we have reviewers that never made a PR???

Yes, e.g. here https://github.com/gammapy/gammapy/pull/3752#discussion_r835538313

Reply:
> Again maybe replace by "opt in" system. Any person could propose to become a Gammapy author, if they think they should be. Decisions are taken by the team on a one by one basis. If a person has done many reviews the decision will be simple, transparent and obvious. However from the other perspective: finding all people and add them to the other list is a lot of work and we will miss those...

I completely agree with you here. People that think should be included in the author list should open a PR to add themselves to the author list, centralized now in the `CITATION.cff`. But at least from the policy, they know they are welcome to do so...

Reply:
We are missing the gammapy CC at this point, no?

Reply:
One trend of opinions was to have the (GitHub) contributors for the code releases and the contributors + people with tasks in the project (e.g. the CC) for the LTS papers.
Should we extend that to the code releases?

Reply:
At least code reviews are very directly connected to code releases